packages/ui/src/components/va-dropdown/VaDropdown.stories.ts

```diff
@@ -1,12 +1,386 @@
+  template: `
+      <va-dropdown
+          keepAnchorWidth
+      >
+        <template #anchor>
+          <div>
+            Short anchor
+          </div>
+        </template>
+        <va-dropdown-content>
+          Loooooooong Dropdown
+        </va-dropdown-content>
+      </va-dropdown>
+      <br/>
+      <va-dropdown
+          keepAnchorWidth
+      >
+        <template #anchor>
+          <div>
+            Loooooooong anchor
+          </div>
+        </template>
+        <va-dropdown-content>
+          Short
+        </va-dropdown-content>
+      </va-dropdown>
+    `,
+})
+
+export const CustomWidth = () => ({
+  components: { VaDropdown, VaValue, VaDropdownContent },
+  template: `
+      <VaValue #default="v" :default-value="true">
+        <button @click="v.value = !v.value">{{ v.value ? 'Short' : 'Long' }}</button>
+        <br/>
+        <va-dropdown placement="bottom-end">
+          <template #anchor>
+            <div>
+              Custom width anchor
+            </div>
+          </template>
+
+          <template #default="{ getAnchorWidth }">
+            <va-dropdown-content :style="{ minWidth: getAnchorWidth() }">
+              {{ v.value ? 'Short' : 'Loooong drooooopdown wwiiiiiidth' }}
+            </va-dropdown-content>
+          </template>
+        </va-dropdown>
+      </VaValue>
+    `,
+})
+
+export const Cursor = () => ({
+  components: { VaDropdown },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          cursor
+      >
+        <template #anchor>
+          <div style="width: 8rem; height: 8rem; border: var(--border-dashed)">
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const CursorWithTarget = () => ({
+  components: { VaDropdown },
+  setup () {
+    return {
+      cursorTarget: ref(null),
+    }
+  },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          :target="cursorTarget"
+          cursor
+      >
+        <template #anchor>
+          <div
+              style="width: 8rem; height: 8rem; border: var(--border-dashed)"
+              ref="cursorTarget"
+          >
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const AnchorSelector = () => ({
+  components: { VaDropdown },
+  template: `
+      <div
+          style="border: var(--border-dashed)"
+          id="anchor-selector"
+      >
+        Click me
+      </div>
+      <va-dropdown
+          anchor-selector="#anchor-selector"
+      >
+        Anchor selector
+      </va-dropdown>
+    `,
+})
+
+export const InnerAnchorSelector = () => ({
   components: { VaDropdown },
-  template: '<VaDropdown/>',
+  template: `
+      <va-dropdown
+          inner-anchor-selector="#inner-anchor-selector"
+      >
+        <template #anchor>
```

Comment:
Need a test case showing that you can only trigger via inner selector and can't trigger by clicking on anything else in #anchor slot.

Might make sense to have 2 divs in slot:
[anchor]
[not anchor]